protos/hub/hub.proto

```diff
@@ -97,6 +99,42 @@ message StatsResp {
   ];
 }
 
+// HubParserService are interface for hub-parser
+service HubParseService {
+  rpc UpdateDiscovery(ParseReq) returns (ParseResp);
+  rpc UpdateScore(ParseReq) returns (ParseResp);
+  rpc UpdateNetwork(ParseReq) returns (ParseResp);
+  rpc Recycle(RecycleReq) returns (ParseResp);
+}
+
+message Candidate {
+  string node_id = 1;
+  uint64 heart_beats = 2;
+  float upload_average = 3;
+  float upload_variance = 4;
+  float download_average = 5;
+  float download_variance = 6;
+}
+
+message ParseReq {
+  repeated Candidate candidates = 1;
+}
+
+message RecycleReq {
+  string index_name = 1;
```

Comment:
where is this from? hard to see where this string can be obtained

Reply:
some kind of id or somewhere from response is better for understanding

Reply:
this is to specify the ES index by its name that need to be recycled

Reply:
will change to enum per discussion